### PR TITLE
Add back to the pipeline deseq2_qc

### LINKED ISSUE #246: Dashes in group identifier
Hej,

thanks for providing the chipseq pipeline! I found a minor issue, either in the documentation or in an R-script:

When having a dash in the group-identifier (e.g. WT_treatment-A ) and by that having a dash in the replicate-ID in the checked design-matrix (e.g. WT_treatment-A_R1_T1 ), the dash is converted to a dot when loading the feature table into an R-script (e.g. featurecounts_deseq2.r ). E.g. heatmaps will then only show multiple "1_R1" as identifier instead of the whole group ID (due to some strsplit I assume), see attached plot.

Changing 
count.table <- read.delim(file=opt$featurecount_file,header=TRUE,skip=1)
into 
count.table <- read.delim(file=opt$featurecount_file,header=TRUE,skip=1, check.names=FALSE)
when loading the table would prevent this. 

Or maybe it's easier to prohibit dashes from the start.
Thanks and best!
Philipp

heatmap

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Add again the process `DESEQ2_QC` to the pipeline (in the DSL1 version `CONSENSUS_PEAKS_DESEQ2`).
Also fixes #246, sample names with dashes displayed as points in the heatmap plot.

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/chipseq branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/chipseq)
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -614,21 +614,27 @@ if (params.macs_gsize) {
...
-    //                     params.narrow_peak? '/narrowPeak' : '/broadPeak',
-    //                     '/consensus/deseq2'
-    //                     ].join('') },
-    //                 mode: 'copy',
-    //                 saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-    //             ]
-    //         }
-    //     }
-    // }
+    if (!params.skip_diff_analysis) {
```
Comment (drpatelh) [inline conf/modules.config:617]: We aren't doing any differential analysis now?

Comment (drpatelh) [inline conf/modules.config:617]:
```suggestion
    if (!params.skip_deseq2_qc) {
```

**Review by drpatelh — APPROVED** [review]
🚀

File: conf/modules.config (hunk 1 above)
Comment (JoseEspinosa) [reply conf/modules.config:617]: Yeap, you are right will drop the parameter for the moment from everywhere...

**Pull request merged**